mrc/localization/color/analyser.py

```diff
@@ -13,14 +13,12 @@ def __init__(self):
         # TODO: Change predicate to fuzzy the hue on hsv circle
 
     def analyse_chunk(self, image_chunk: ndarray) -> dict:
-        pixel_list = image_chunk.reshape(image_chunk.shape[0] * image_chunk.shape[1], 3)
+        chunk_shape = image_chunk.shape
```

Comment:
This shall come on the level of review. Please unstage those changes I'll change them in the branch I am pushing right now. img_chunk.shape ~ O(1). I don't feel like it's necessary to add a var here.